src/components/BottomBar/BottomBarCourse.vue

```diff
@@ -308,11 +308,6 @@ export default Vue.extend({
       this.isSmallerWidth = window.innerWidth <= 976;
     },
 
-    toggle() {
-      // @ts-ignore
-      this.$emit('toggle', this.isExpanded);
-    },
-
```

Comment:
this function is unused, and `isExpanded` is not in data or props. It's likely leftover dead code, so i removed it.